### DISCUSSION TITLE
[npmjs.com bug] "Fund this package" button only works for the first `package.json#funding` entry

### DISCUSSION THREAD

**Discussion by broofa** [community/community#128438] [npm]
In https://github.com/npmgraph/npmgraph/pull/134 I added @fregante and I (@broofa) to the `package.json#funding` field.

**Expected behavior**:  On the https://www.npmjs.com/package/npmgraph page there should be some way for people to fund either of us, or both of us.

**Actual behavior**: Clicking the "Fund this package" button takes you directly to the first URL listed in the `funding` field, with no indication that there are additional funding options available.

**Suggested solution**:  Instead of a "Fund this package" button, have a "Fund this package" section in the sidebar with buttons for each funding option.  Ideally, the interface would recognize popular funding URL formats (e.g. GitHub's `.../sponsors/:githubId`) and provide a corresponding badge/button. For example:

<image width="300" src="https://user-images.githubusercontent.com/164050/198840183-3421d2fa-3916-4088-b3e6-f227b1d7eb02.png" />

**Comment by monishcm** [comment]
Thanks for the feedback. We will evaluate this.
